### DISCUSSION TITLE
Student benefits - Student card contains the name on the front and the expiration date on the back.

### DISCUSSION THREAD

**Discussion by Samina992** [community/community#136133] [GitHub Education]
### Hi everyone,

Hi, i am trying to apply for github student benefits, and my university student card features my name on the front side of the card, and the expiration date on the back side. Now, I want to take the picture of the student card using github student benefits application form integrated camera feature, and there can only be 1 image. What is the possible solution for it?

**Comment by AEdgecombe** [comment]
Here’s a possible solution to handle the issue:

You can take photos of both the front and back of your student ID and **merge them into a single image**. Here's how you can do it:

1. **Take Photos of Both Sides**: Use your phone or camera to take clear pictures of both the front and back sides of your student card.

2. **Use an Image Editing Tool**: You can use a free image editor like:
   - **Canva**
   - **Microsoft Paint**
   - **Preview (on Mac)**
   - **Google Docs**
   - Any photo collage app

   Place the two images side by side (or one above the other) in the same canvas to create a single image file. Ensure that both your name and expiration date are visible.

3. **Upload the Merged Image**: Once you've combined the images into one file, save it as a JPEG or PNG and upload it as your verification image.

This way, you satisfy the requirement of showing both sides of your student ID while sticking to the one-image upload limit.

**Comment by AnshulJagotra** [comment]
When applying for GitHub student benefits and faced with the requirement to submit only one image that includes both the front side with your name and the back side with the expiration date of your student card, you can follow these steps:

Capture Clear Photos: Using a smartphone or camera, take separate, clear photos of both the front and back sides of your student ID card.

Use a Document Scanner App: Install a document scanner app on your smartphone, such as Adobe Scan, CamScanner, or Google Drive’s built-in scanner. These apps allow you to capture and merge multiple images into a single PDF or image file.

Merge Images: Within the document scanner app, select the option to create a new document or combine images. Choose the images you've captured of both sides of your student card. Adjust and align them so that both your name (from the front side) and the expiration date (from the back side) are clearly visible in the final merged document.

Save and Upload: Once you’ve merged the images, save the resulting file as a PDF or JPEG. Ensure the merged file is clear and readable. This single file will now contain both required sides of your student ID.

Submit the File: Proceed to upload this merged file as your verification image in the GitHub student benefits application form. This approach meets the requirement of submitting only one image while including all necessary information from both sides of your student card.

By following these steps, you can effectively comply with the application requirements without needing to physically alter your student card or face limitations related to the single-image upload restriction.
